src/lib.rs

```diff
@@ -188,6 +188,10 @@ pub struct InitOptions {
 pub fn init(options: &InitOptions) -> Result<ZWaveManager,()> {
     let mut ozw_options = try!(options::Options::create("./config/", "", "--SaveConfiguration true --DumpTriggerLevel 0 --ConsoleOutput false"));
 
+    try!(options::Options::add_option_int(&mut options, "PollInterval", 60000)); //enable polling each 60 seconds
+    try!(options::Options::add_option_bool(&mut options, "IntervalBetweenPolls", true));
+    try!(options::Options::add_option_bool(&mut options, "ValidateValueChanges", true));
+    try!(options::Options::add_option_bool(&mut options, "Associate", true));
```

Comment:
I'll plead guilty to just copying from the domoticz code.

It looks like the default value of Associate is already true, so we can remove that line:
https://github.com/OpenZWave/open-zwave/blob/master/cpp/src/Options.cpp#L110

It looks like ValidateValueChanges must be old. I found references in MinOZW app (where it sets it to true), but not anyplace that references it.

So I'll remove those 2.